### PR TITLE
BUGZ-141: Fix the LOD target value not in sync with the RefreshRate setting

### PR DESCRIPTION
**Pull request by samcake**
This pr Fixes the case when the LOD render target is ABOVE the refresh rate current setting.
THis avoids the LOD Angle to go wide when the Application is being throttled intentionally by the refreshrate manager.

 https://highfidelity.atlassian.net/browse/BUGZ-141

Test plan and technical details found in Jira

### CODE REVIEW

File: libraries/display-plugins/src/display-plugins/RefreshRateController.cpp [hunk 1]
```diff
@@ -30,7 +30,7 @@ int RefreshRateController::getRefreshRateLimitPeriod() const {
     return durationNanosecondsToHz(_refreshRateLimitPeriod);
 }
 
-void RefreshRateController::sleepThreadIfNeeded(QThread* thread, bool isHmd) {
+std::chrono::nanoseconds RefreshRateController::sleepThreadIfNeeded(QThread* thread, bool isHmd) {
```
Comment (ZappoMan) [inline libraries/display-plugins/src/display-plugins/RefreshRateController.cpp]: what is this change for?

Comment (ZappoMan) [reply libraries/display-plugins/src/display-plugins/RefreshRateController.cpp]: asked and answered

Comment (samcake) [reply libraries/display-plugins/src/display-plugins/RefreshRateController.cpp]: to use that in the future

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19482/HighFidelity-Beta-PR15566-04c139c.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19482/HighFidelity-Beta-Interface-PR15566-04c139c.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19482/HighFidelity-Beta-PR15566-04c139c.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19482/HighFidelity-Quest-Beta-PR15566-04c139c.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19482/HighFidelity-Beta-PR15566-04c139c.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19482/HighFidelity-Beta-Interface-PR15566-04c139c.exe

**Pull request merged**